config/backend-config/admin.go

```diff
@@ -19,7 +19,7 @@ func (bca *BackendConfigAdmin) RoutingConfig(filterProcessor bool, reply *string
 		}
 	}()
 
-	outputJSON := GetConfig()
+	outputJSON := DefaultBackendConfig.GetConfig()
```

Comment:
Aren't we at the same package? Can we unexport the `GetConfig` method?

Reply:
Oh yes, good point, I think I can remove it now, thanks :+1: